### PR TITLE
add: sect of force buff

### PR DESCRIPTION
**Pull request by JayAndromeda**
will need to add support in foundry-ui as a part of this change.

### CODE REVIEW

**Comment by JayAndromeda** [comment]
@gothfemme - i gotchu girl

SectOfForce = 3268255645,

{
    hash: AbilityBuffs.SectOfForce,
    label: "Sect of Force",
    classType: DestinyClassType.Neutral,
    subtitle: "Aeon Cult Mod",
    iconSrc:
      "https://www.bungie.net/common/destiny2_content/icons/9a4852af05d48278c6dc206a282d5460.jpg",
 },

**Review by BarmonHammer — APPROVED** [review]

**Comment by JayAndromeda** [comment]
welp both this and the sect of insight perk we have added already are being changed next season !
source: https://www.bungie.net/7/en/News/article/this-week-in-destiny-11-22-23

**Comment by oh-yes-0-fps** [comment]
This should probably just be updated and merged to get it outa the way, I can do it later today.

[compendium](https://docs.google.com/spreadsheets/d/1WaxvbLx7UoSZaBqdFr1u32F2uWVLo-CJunJB4nlGUE4)

File: src/perks/buff_perks.rs [hunk 1]
```diff
@@ -421,19 +421,43 @@ pub fn buff_perks() {
             }
         }),
     );
-    add_dmr(
-        Perks::NoBackupPlans,
-        Box::new(|_input: ModifierResponseInput| -> DamageModifierResponse {
-            if *_input.calc_data.weapon_type != WeaponType::SHOTGUN || _input.value == 0 {
-                return DamageModifierResponse::default();
```
Comment (FlaminSarge) [inline src/perks/buff_perks.rs:428]: This removal doesn't seem intended for this PR?

**Pull request merged**